contracts/interfaces/IEscrow.sol

```diff
@@ -1,7 +1,9 @@
 import { LoanLib } from "../lib/LoanLib.sol";
 import { IModule } from "./IModule.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 interface IEscrow is IModule {
 
   function healthcheck() external returns (LoanLib.STATUS status);
```

Comment:
Just FYI but this is already defined in https://github.com/debtdao/smart-contracts/blob/d5767ff7f93536ee91a470be5e66f3d983033d5c/contracts/interfaces/IModule.sol#L4